dotnet/src/SemanticKernel.Core/Functions/KernelFunctionFromMethod.cs

```diff
@@ -281,6 +288,7 @@ ValueTask<FunctionResult> Function(Kernel kernel, KernelFunction function, Kerne
             Function = Function,
             Name = functionName!,
             Description = method.GetCustomAttribute<DescriptionAttribute>(inherit: true)?.Description ?? "",
+            Attributes = [.. Attribute.GetCustomAttributes(method, inherit: true)],
```

Comment:
This only extracts attributes off the method. Possibly fine for a first pass but there's potentially value in also grabbing attributes off the plugin itself, parameters, etc.